python/sqlflow_submitter/xgboost/__init__.py

```diff
@@ -10,3 +10,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from sqlflow_submitter.xgboost import feature_column
+
+
+class DataTypeCollection(object):
```

Comment:
We can use this style:
```python
class DataType(object):
    INT64 = "int64"
    FLOAT32 = "float32"
    STRING = "string"
```

Reply:
It may not work. See [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/pkg/sql/codegen/codegen_feature_column.go#L25). If we want to unify the codegen of TensorFlow and XGBoost, we must support data type style like `xgboost.dtypes.float32` (something corresponds to `tf.dtypes.float32`) instead of `xgboost.DataType.FLOAT32`.